### PR TITLE
remove footer items

### PR DESCRIPTION
**Pull request by toufali**
https://mozilla-hub.atlassian.net/browse/MZTDN-154

### CODE REVIEW

**Comment by aaga** [comment]
Shouldn't we be doing something more similar to what we have on mozilla.social? i.e. Keep the footer but add our own links for privacy policy and terms of service? Otherwise if/when we make Elk the default front-end, those links will no longer be accessible.

Screen Shot 2023-06-15 at 3 26 43 PM

**Comment by toufali** [comment]
> Shouldn't we be doing something more similar to what we have on mozilla.social (see below; we could simplify)? i.e. Keep the footer but add our own links for privacy policy and terms of service? Otherwise if/when we make Elk the default front-end, those links will no longer be accessible.
> Screen Shot 2023-06-15 at 3 26 43 PM

@aaga Yes! Good call.  It will require building out the Privacy Policy and Terms pages, I believe?  I can create a separate ticket for that, and we can add under separate PR assuming you agree.

**Comment by aaga** [comment]
> > Shouldn't we be doing something more similar to what we have on mozilla.social (see below; we could simplify)? i.e. Keep the footer but add our own links for privacy policy and terms of service? Otherwise if/when we make Elk the default front-end, those links will no longer be accessible.
> > Screen Shot 2023-06-15 at 3 26 43 PM
> 
> @aaga Yes! Good call. It will require building out the Privacy Policy and Terms pages, I believe? I can create a separate ticket for that, and we can add under separate PR assuming you agree.

Would it be too weird to just use the links for Privacy Policy and Terms that we have now? They live on the mozilla.social domain already, and given that we are (most likely) merging everything into the same domain, we can change the formatting / structure of those pages at a later date when we see fit? Just seems extraneous to me to have two versions of each of those pages, I guess.

**Comment by toufali** [comment]
> Would it be too weird to just use the links for Privacy Policy and Terms that we have now? They live on the mozilla.social domain already, and given that we are (most likely) merging everything into the same domain, we can change the formatting / structure of those pages at a later date when we see fit? Just seems extraneous to me to have two versions of each of those pages, I guess.

@aaga but those privacy/terms pages load into the Mastodon front-end...  I'd be concerned a user clicks that and gets stuck in mastodon-world.  I suppose we could just add the links for now, and follow up with a second PR that actually builds those pages into Elk.  Is that what you're thinking?

**Comment by aaga** [comment]
> @aaga I added privacy/terms links, but as noted above, they are in the Mastodon FE. I can create a new ticket to incorporate into Elk.

Yes that sounds good!

In my original comment, I was also suggesting to:
1) keep some of the Elk stuff around (like About and Sponsor, so that we're not claiming Elk to be our own)
2) keep a way to see the currently deployed commit / version (e.g. via the "View Source Code" link we have in Mastodon)

I feel pretty strongly about 1 and less strongly about 2. Maybe we discuss this on Monday.

File: components/nav/NavFooter.vue [hunk 1]
```diff
@@ -28,68 +22,14 @@ function toggleDark() {
...
-      &middot;
-      <NuxtLink href="https://chat.elk.zone" target="_blank" external>
-        Discord
+      <NuxtLink href="https://mozilla.social/privacy-policy" target="_blank" external>
+        Privacy Policy
       </NuxtLink>
       &middot;
-      <NuxtLink href="https://github.com/elk-zone/elk" target="_blank" external>
-        GitHub
+      <NuxtLink href="https://mozilla.social/terms" target="_blank" external>
```
Comment (bakulf) [inline components/nav/NavFooter.vue:31]: I wonder if we can make these section "programmable" via the config file. In this way, we can uplift this PR too.

**Review by bakulf — APPROVED** [review]

**Pull request merged**